topi/python/topi/generic/extern.py

```diff
@@ -35,8 +35,12 @@ def schedule_extern(outs):
     sch: Schedule
         The computation schedule for the op.
     """
-    target = tvm.target.current_target(allow_none=False)
-    if target.target_name != "llvm":
-        raise RuntimeError("schedule_extern not registered for '%s'" % target)
     outs = [outs] if isinstance(outs, tvm.tensor.Tensor) else outs
-    return tvm.create_schedule([x.op for x in outs])
+    s = tvm.create_schedule([x.op for x in outs])
+
+    tvm.schedule.AutoInlineInjective(s)
+    for out in outs:
+        if isinstance(out.op, tvm.tensor.ExternOp):
+            continue
+        _schedule_injective(out.op, s)
```

Comment:
@vinx13, I have moved this logic from `cuda/extern.py` to `generic/extern.py`. Will `_schedule_injective` still call the correct overridden function per target? Or will this just call the default one?

If it just calls the default one, it seems like I have to add a new file, `x86/extern.py`